docs/docs/03-main-concepts.md

```diff
@@ -56,21 +55,21 @@ After Snowpack builds your dependencies, any package can be imported and run dir
 
 Snowpack supports JSX & TypeScript source code by default. You can extend your build even further with [custom plugins](#build-plugins) that connect Snowpack with your favorite build tools: TypeScript, Babel, Vue, Svelte, PostCSS, Sass... go wild!
 
-
 ### Snowpack's Build Pipeline
 
 ![build output example](/img/snowpack-build-example.png)
 
 `snowpack build` - When you're ready to deploy your application, run the build command to generate a static production build of your site. Building is tightly integrated with your dev setup so that you are guaranteed to get a near-exact copy of the same code that you saw during development.
 
-
 ### Bundle for Production
```

Comment:
1. feels like this section is getting pretty long, can we shorten / remove the parts that repeat?
2. is this larger section now "Optimize for Production"
3. the voice these headers taking is describing the job you're looking to do. Can the subheaders match that?

#### Example:

- Optimize for Production
  - With a Bundler
  - Without a Bundler

Reply:
I’m +1 for any docs changes necessary, but WDYT about getting it merged based on code first, and doing the docs rev in a followup (I could even revert the docs changes for now)? I’ve already had to rebase this PR many times from the other merges, and it looks like I have another merge conflict to resolve this morning.